01-Login/config.xml

```diff
@@ -82,19 +82,18 @@
     <plugin name="cordova-plugin-whitelist" spec="1.3.1" />
     <plugin name="cordova-plugin-device" spec="1.1.4" />
     <plugin name="cordova-plugin-splashscreen" spec="~4.0.1" />
-    <plugin name="cordova-plugin-ionic-webview" spec="^1.1.11" />
     <plugin name="cordova-sqlite-storage" spec="^2.2.0" />
     <allow-navigation href="http://192.168.42.127:8100" />
     <allow-navigation href="http://192.168.254.18:8100" />
-    <engine name="ios" spec="4.5.4" />
     <plugin name="cordova-plugin-customurlscheme" spec="^4.3.0">
         <variable name="URL_SCHEME" value="com.auth0.ionic" />
         <variable name="ANDROID_SCHEME" value="com.auth0.ionic" />
-        <variable name="ANDROID_HOST" value="kmaida.auth0.com" />
```

Comment:
I noticed that Kim's Auth0 domain also appears in the `package.json` file, we should probably find a way to remove that.

Reply:
I see two options here:
1. Update `package.json` like:
```js
{
// ...
  "cordova-plugin-customurlscheme": {
        "URL_SCHEME": "com.auth0.ionic",
        "ANDROID_SCHEME": "com.auth0.ionic",
        "ANDROID_HOST": "{DOMAIN}",
        "ANDROID_PATHPREFIX": "/cordova/com.auth0.ionic/callback"
      },
}
```
This way will be properly filled when download from docs, but we should add instructions in repo's README to replace the domain.

2. Remove the plugin from the sample and add the instructions in README to install/configure the plugin.

Reply:
@alexisluque Let's go with option 1 and put some more instructions into the tutorial to update this file.